### DISCUSSION TITLE
lit-html highlighting

### DISCUSSION THREAD

**Discussion by LucasRitossa** [fitrh/init.nvim#7] [General]
Hey I noticed on a Neovim thread you seemed to have syntax highlighting for lit html templates (e.g. html``). How exactly is this achieved as I cannot seemed to get this result using Treesitter?

**Comment by fitrh** [comment]
Have you installed the `tsx` parser? All I use is treesitter with `javascript`, `typescript` and `tsx` parser

**Reply by LucasRitossa** [reply]
I still didn't get the highlighting with those parsers but luckily :TSInstall all seemed to do the trick. Not sure what parser ended up fixing it, but I suppose that doesn't matter. Thank you :)
